Acme/Certificate/CertificateMetadata.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+/*
+ * This file is part of the ACME PHP library.
+ *
+ * (c) Titouan Galopin <galopintitouan@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace AcmePhp\Bundle\Acme\Certificate;
+
+use AcmePhp\Core\Ssl\Certificate;
+
+/**
+ * Entity representing a certificate.
+ *
+ * @author Jérémy Derussé <jeremy@derusse.com>
+ */
+class CertificateMetadata
```

Comment:
Do you think about the merge function ? or Setters ?

For the first one, I may replace it by a static function which return a new immutable instance (does it worst it ?)
For the second, it's a mistake, I'll fix it